### DISCUSSION TITLE
LDAP use login user as bind user

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#12729] [Archive]
Our customer has a LDAP configuration where they don't have any lookup user that we can use as a bind user. :(

The issue is that its a larger company and we only do a cms for some small part of it and they don't want to introduce a lookup user that we can use.

So we would need the ability to use the login user as a bind user to fetch the necessary stuff.
But I don't see how this works with directus based on what the docs say.

So is this somehow possible?

**Comment by aidenfoxx** [comment]
So LDAP was originally coded with support for using the authenticating user as the bind user, but it was confusing because if you use the authenticating user as the bind user, you can't use alternative attributes (other than the CN) as the identifier, and you can't log in users from sub-directories. So in the end I removed that ability.

There is no plan to support this in the future.
